src/index.js

```diff
@@ -423,6 +430,47 @@ function setCookiesAndTokens(tokens) {
   setCookie(tokens.id.value, tokens.id.cookieOptions, "id");
   setCookie(tokens.refresh.value, tokens.refresh.cookieOptions, "refresh");
   setTokensFromCookies();
+  setUser();
+}
+
+/**
+ * Define user attributes & methods based on access & ID token.
+ */
+function setUser() {
+  if (!store.idToken) {
+    throw new Error("ID token has not been set.");
+  }
+  if (!store.accessToken) {
+    throw new Error("Access token has not been set.");
+  }
+
+  const decodedIdToken = jwt.decode(store.idToken);
+  const decodedAccessToken = jwt.decode(store.accessToken);
+
+  store.user = {
+    ...user({ idToken: decodedIdToken, accessToken: decodedAccessToken }),
+    ...apiResource({
+      store,
+      path: "/users",
+      id: decodedAccessToken.userId,
+      afterUpdate: refresh,
+    }),
+  };
+}
```

Comment:
Here is where `Userfront.store.user` is created.

- `...user` is the user information from the tokens
- `...apiResource` is what provides the `user.update` method
  The `afterUpdate` callback option was created in order to keep the `refresh` in one place and leave room to do other work if necessary.